src/main/java/com/rylinaux/plugman/util/BukkitCommandWrap.java

```diff
@@ -25,139 +31,81 @@ public class BukkitCommandWrap {
     private Constructor bukkitcommandWrapperConstructor;
 
     public BukkitCommandWrap() {
-        try {
-            this.nmsVersion = Bukkit.getServer().getClass().getPackage().getName().replace(".", ",").split(",")[3];
-        } catch (ArrayIndexOutOfBoundsException e) {
-            e.printStackTrace();
-            this.nmsVersion = null;
+        String prefix = cbPrefix();
+        String[] packageParts = prefix.split("\\.");
+        String versionPart = packageParts[packageParts.length - 1];
+        this.craftBukkitVersioning = versionPart.startsWith("v1_");
+        if (this.craftBukkitVersioning) {
+            this.nmsVersion = versionPart;
         }
+
+        boolean nmsVers = true;
+        try {
+            Class.forName("net.minecraft.server.MinecraftServer");
+            nmsVers = false;
+        } catch (ClassNotFoundException ignore) {}
+        this.nmsVersioning = nmsVers;
```

Comment:
Why do it like this?

Couldn't we just set it to false by default and set it to true, if we encounter the `ClassNotFoundException`?

Reply:
Shortest / cleanest code, nothing else to the reasoning for me